### ISSUE TITLE
recherche avancée

### ISSUE THREAD

**Issue by mnauge** [Francoralite/francoralite#197]
s'inspirer de ça : https://isidore.science/as

**Comment by Marion-L1** [comment]
Liste de cas d'usage
[Cas d'usage.odt](https://github.com/Francoralite/francoralite/files/7146353/Cas.d.usage.odt)

**Comment by mnauge** [comment]
@Marion-L1 @lluc commence à se pencher sérieusement sur la question de la recherche avancée. Peux-tu confirmer que le document Cas d'usage.odt ci-dessus est toujours d'actualité ? Souhaites-tu y apporter des modifications ? Il va en effet servir de base pour le "design" de la recherche avancée.
Par avance merci.

**Comment by lluc** [comment]
@Marion-L1 Le tableau qui figure sur la dernière page du document (Cas d'usage.odt)  a quelle fonction ? Que doit-on faire avec ? Est-il une synthèse des pages précédentes ?

**Comment by Marion-L1** [comment]
@lluc @mnauge Oui, le document me parait encore d'actualité. Le tableau de la dernière page correspondait en effet à la synthèse des pages précédentes, donc aux besoins de recherches croisées pour atteindre le but du chercheur en fonction de ses attentes.

**Comment by mnauge** [comment]
Pour alimenter nos réflexions que pensez-vous (@Marion-L1 ) de ce genre de recherche : 
https://transcrire.huma-num.fr/s/transcrire/recherche?q=noce

essayer de cliquer à gauche sur Discipline : Ethnologie (28)
ça amene à une page où ce filtre est conservé.
puis cliquer à gauche sur Projet:   Carnets d'Antoine d'Abbadie (5)
ça amene à une page où ce filtre est conservé (et les filtres précédents).
puis cliquer à gauche sur Statut : Terminé (2)

@lluc @fab2713 regarde la tête de l'url au fur et à mesure et dis moi d'un point de vue end-point si ça te semble envisageable sans s'embarquer dans un truc trop complexe ?

**Comment by lluc** [comment]
@mnauge  d'un point de vue end-point, cela me semble envisageable.  
... reste à voir comment l'intégrer dans DRF.

**Comment by mnauge** [comment]
@lluc , c'est quoi  DRF ?  

Pour faire avancer la question des spécifications plus précises, j'ai une réunion virtuelle avec @Marion-L1 et Marlène lundi 20 à 11h. Si tu/vous êtes dispo. ça peut être intéressant de participer aux discussions :-)

**Comment by lluc** [comment]
@mnauge  DRF : Django Rest Framework

**Comment by mnauge** [comment]
@lluc pour info. en parcourant un document de @Marion-L1 au sujet de l'amélioration de l'UIX, elle avait noté : 
Ajouter des champs à la recherche avancée : Rendre opérationnelle pour les champs : informateur / enquêteur / cote / mot-clé / lieu / langue / date / domaine / genre / fonction / danse / timbre / cote du timbre / incipit / refrain / coupe / référence / instrument / type de voix/instrument / civilité / Niveau d'information
